alpha/apps/kaltura/lib/kSimuliveUtils.php

```diff
@@ -340,6 +341,10 @@ public static function getSourceDurations($sourceEntries, $event)
 				$durations[] = $entryRoundedDuration;
 			}
 		}
+		if ($postEndDurationMs)
+		{
+			$durations[count($durations) - 1] = $postEndDurationMs;
```

Comment:
it can't happen, as if there is a postEndDurationMs - it means that at least one entry exists in "sourceEntries" array (the postEndEntry), and the "durations" array length has the same length as "sourceEntries" (as we insert a duration for each entry in sourceEntries - in the for loop above)